Sources/TuistCache/GraphMappers/CacheMapper.swift

```diff
@@ -25,9 +25,9 @@ public class CacheMapper: GraphMapping {
 
     // MARK: - Init
 
-    public convenience init(config: Config) {
+    public convenience init(config: Config, cloudClient: CloudClienting) {
```

Comment:
What about not exposing this from the constructor. If there's a need for dependency injection at this level?